internal/config/uplift.go

```diff
@@ -47,6 +47,7 @@ type Uplift struct {
 	GitLab        *GitLab       `yaml:"gitlab" validate:"omitempty"`
 	Hooks         *Hooks        `yaml:"hooks" validate:"omitempty"`
 	Env           []string      `yaml:"env" validate:"dive,min=1"`
+	DirtyFiles    []string 		`yaml:dirtyFiles validate:"omitempty"`
```

Comment:
Have you installed the Go Language Server (Go Plugin if using VSCode)? The formatting is invalid in this file

Reply:
had to reinstall it :( 
plugin is working now